web/src/components/atoms/core/Table.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react';
```

Comment:
Are these in for future use?

Reply:
Yeah, see description.

> Base components are redefined to make it clear which atomic components are used by the application (a bit overkill atm, but will scale out nicely, I think).

By redefining all the core components we use, it gives us a nice clear view of all the atoms our app is built on. These will change a bit (to include app styles etc.), so it's also useful for future usage.